pyproject.toml

```
@@ -85,8 +85,12 @@ pinned = [
 ]
 
 conda_pinned = ["fenics-dolfinx==0.9.0", "freecad==1.0.0"]
-openmc = ["openmc>=0.14.0", "openmc_data==2.3.1"]
 process = ["process @ git+https://github.com/ukaea/process@v3.2.0"]
+openmc = [
+    "openmc>=0.14.0",
+    "openmc_data==2.3.1",
+    "tokamak-neutron-source @ git+ssh:github.com/Fusion-Power-Plant-Framework/tokamak-neutron-source",
```

Once the package is opened up (before we have a version)
```suggestion
    "tokamak-neutron-source @ git+https://github.com/Fusion-Power-Plant-Framework/tokamak-neutron-source@main",
```